test/transactionRejection.js

```diff
@@ -138,7 +133,7 @@ describe("Transaction rejection", function() {
     };
 
     // don't send with web3 because it'll inject its own checks
-    provider.send(request, (_, response) => {
+    provider.send(request, async(_, response) => {
```

Comment:
promisify this.

Reply:
Note: Created an issue for this https://github.com/trufflesuite/ganache-core/issues/351